storage/postgres/notification_storage.go

```diff
@@ -69,3 +79,38 @@ func (ns *notificationStorageImpl) GetNotification(ctx context.Context, id strin
 	}
 	return notificationObj.(*types.Notification), nil
 }
+
+func (ns *notificationStorageImpl) ListNotifications(ctx context.Context, platformID string, from, to int64) ([]*types.Notification, error) {
+	// TODO: replace with less than or equal operator when available
+	listQuery1 := query.ByField(query.GreaterThanOperator, "revision", strconv.FormatInt(from, 10))
+	listQuery2 := query.ByField(query.LessThanOperator, "revision", strconv.FormatInt(to+1, 10))
+
+	filterByPlatform := query.ByField(query.EqualsOrNilOperator, "platform_id", platformID)
+	objectList, err := ns.storage.List(ctx, types.NotificationType, listQuery1, listQuery2, filterByPlatform)
+	if err != nil {
+		return nil, err
+	}
+	notificationsList := objectList.(*types.Notifications)
+	// TODO: Should be done in the database with order by
```

Comment:
The same - create a ticket/issues for the desired/missing functionalities and clean the TODOs.